### DISCUSSION TITLE
多语言会在未来开发计划吗

### DISCUSSION THREAD

**Discussion by dingwen1** [spacedriveapp/spacedrive#1668] [Feature Request]
想问问，未来会开发多语言版本吗？中文可以吗

**Comment by brxken128** [comment]
Yes, we plan to add full support for as many languages as we viably can. Currently we're in the early alpha stages, and many components of the app change every single day - keeping on top of translations at the moment would be almost impossible.

However, once things stabilize, translations can start and we aim to support as many languages as we can in order to ensure accessibility across the globe. Quite a few members of the team are multi-lingual and I'm sure they wouldn't mind helping out as/when they can, too.
